scripts/start-cbt.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# set -e: exit asap if a command exits with a non-zero status
+set -e
+
+echoerr() { printf "%s\n" "$*" >&2; }
+
+# print error and exit
+die () {
+  echoerr "ERROR: $1"
+  # if $2 is defined AND NOT EMPTY, use $2; otherwise, set to "160"
+  errnum=${2-160}
+  exit $errnum
+}
+
+# Required params
+[ -z "${CBT_USERNAME}" ] && die "Required env var CBT_USERNAME"
+[ -z "${CBT_AUTHKEY}" ] && die "Required env var CBT_AUTHKEY"
+[ -z "${CBT_TUNNEL_ID}" ] && die "Required env var CBT_TUNNEL_ID"
+
+
+# Start tunnel
+cbt_tunnels \
```

Comment:
Where does this file come from? Is this a binary? Should be added to the Dockerfile, right?

Reply:
The file is a node module. We do have a binary, should it be used instead?

Reply:
If you check the Dockerfile, that is what is done for Sauce Labs, BrowserStack and TestingBot... They all provide a binary that can be started for the tunnel. A node module would be an overkill because we would need to install node in the Zalenium image.